SharpToken.Benchmark/README.md

```diff
@@ -0,0 +1,434 @@
+# SharpToken Benchmark
+
+This runs a encode benchmark over all samples in GptEncoderTestSamples.txt
```

Comment:
This file shows a history of benchmarks. It shows how the changes effect performance.